website/docs/reference/deprecations.md

```diff
@@ -2,7 +2,13 @@
 title: "Deprecations"
 ---
 
-As dbt runs, it generates different categories of [events](/reference/events-logging), one of which is _deprecations_. Deprecations are a special type of warning that lets you know that there are problems in parts of your project that will result in breaking changes in a future version of dbt. It is important to resolve any deprecation warnings in your project before the changes are made.
+:::note
+
+"Deprecated functionality" means that the functionality still works today, but only for historical purposes and it will be removed in a future version.
```

Comment:
```suggestion
Deprecated functionality still works in the v1.10 release, but it is no longer supported and will be removed in a future version.  

This means the deprecated features only present a warning but don't prevent runs and other commands (unless you've customized the behavior). 

When the functionality is eventually removed, it will cause errors in your dbt runs after you upgrade if the deprecations are not addressed.

```